flathub.json

```diff
@@ -0,0 +1,3 @@
+{
+  "only-arches": ["x86_64"]
```

Comment:
that's disappointing. Why no aarch64 support? You say it's focused on multi-platform, this contradict the statement.

Reply:
I expected that comment :) , but Rome isn't built in a day. For this first release of the project I focus on Linux x86_64 and Android. aarch64 may come later.

Reply:
why not building from source here?

Reply:
Community Remote is a Flutter app, that brings the multi platform support. I took another registered Flutter app as an example: https://github.com/flathub/com.enjoyingfoss.feeel.

Reply:
Rome wasn't build out of sand castles either.